.github/workflows/test.yaml

```diff
@@ -888,23 +888,6 @@ jobs:
           spread -discard -reuse-pid="$(echo "$r" | grep -o -E '[0-9]+')";
         done
 
-    - name: report spread errors
```

Comment:
how can we make sure not to forget to leave this out when we merge back?

Reply:
hm I suppose leaving `# TODO placeholder for spread error reporting` should cause a conflict

Reply:
There will be a conflict. Because this is modified in master.